firebase-storage-app/firebase-storage-app.gradle

```diff
@@ -14,6 +14,7 @@
 
 
 apply plugin: "com.android.application"
+apply plugin: 'com.google.gms.google-services'
```

Comment:
@vkryachko : With this line, the test app runs fine, but `:check-changed` failed because it can't find the PLIST file. Do you have any suggestions for me at this point?

Reply:
Hey, so the issue is that until now all sdks and apps in the repo had the following requirements to run their tests:
1. unit tests - don't need `google-services.json`
2. instrumentation tests - need `google-services.json`

For this reason our CI only "mounts" the json file for `device-check` and `connected-check` jobs. And the way gradle is configured is to only apply the `google-services` if it detects that instrumentation tests.

I am also a bit confused and unclear about what the intent is for this app, you previously mentioned that it is supposed to only be run manually, not in CI, and here you are trying to plumb it to make CI pass. It might make sense to take a step back and chat about it more to get on the same page, wdyt?

Reply:
You are correct. Ideally, this app would not run on CI. It has no actionable tests. So maybe the question here is how I can exclude it from `:check-changed`. Let me schedule a meeting to help clarify :)